direct/data/h5_data.py

```diff
@@ -129,12 +129,13 @@ def __init__(
         else:
             self.logger.info("Using %s h5 files in %s.", len(filenames), self.root)
 
+        self.sensitivity_maps = cast_as_path(sensitivity_maps)
+
```

Comment:
Can we not start assuming the objects are all pathlib.Path?

Reply:
Hmm the above might be tricky as it's an input from the yaml file.

Reply:
Btw the above was old code, I just moved it a few lines back.  And I added some more functionality for when sensitivity maps are read from a file.